spec/order_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'order'
+describe Order do
+  let(:dummy_order) {double :dummy_order}
+  let(:test_order) {"2 carbonara, 1 tiramisu and I dunno"}
+  let(:menu_choice) {{ Mains: 'Price', Margherita: 13, Pasta: 12, Risotto: 11, Calzone: 10, Carbonara: 12, Bolognese: 12, Gnocchi: 11 }}
```

Comment:
Metrics/LineLength: Line is too long. [136/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)